### DISCUSSION TITLE
char jump match highlight

### DISCUSSION THREAD

**Discussion by MunifTanjim** [folke/flash.nvim#77] [Q&A] [answered]
I've set `FlashLabel` to `{ bg = color.purple, fg = color.bg, italic = true }`.

Now search (`require("flash").jump()` or `/`) is highlighted as I wanted:

![image](https://github.com/folke/flash.nvim/assets/8050659/d0f9f4e0-0cfe-4d16-a465-67d8ca5cf9ba)

But for char jump (`f`, `F`, `t`, `T`), it's a bit unexpected:

![image](https://github.com/folke/flash.nvim/assets/8050659/b176bc9a-009f-41c4-9014-16bb9a236eff)

I think those shouldn't be highlighted with `FlashLabel`? They're just matches, so should be highlighted with `FlashMatch`?

Didn't create an issue in case this was somehow the desired behavior. If it's the desired behavior, is there any way to configure it as I wanted above?

**Comment by MunifTanjim** [comment]
Changing

https://github.com/folke/flash.nvim/blob/85a12eaaa7f7c8b903b5480113434ab773d94452/lua/flash/highlight.lua#L118-L118

to

```lua
          hl_group = state.opts.highlight.groups.match,
```

solves it for me.

**Reply by folke** [reply, marked as answer]
This is how it is intended.

You can change it in the config for `modes.char.highlight.groups`. check the config section in the readme
